### DISCUSSION TITLE
Happy New Year 2023

### DISCUSSION THREAD

**Discussion by tasszz2k** [netease-im/camellia#94] [General]
Happy Lunar New Year! I wish that in this new year, we can accomplish all of our plans and see improvement in the camellia project.

Special thanks to @caojiajun 
: D 
🎆 🎆 🎆

**Comment by caojiajun** [comment]
Happy New Year to all those who care about `camellia`! 
Special thanks to those who contribute to `camellia`, @tasszz2k @yangxb2010000 @tain198127 @chanjarster @zhaoxiaojie0415 @HongliangChen-963 @phuc1998 

🎉🎉🎉
